ui/lib/core/addon/components/empty-state.js

```diff
@@ -13,6 +13,7 @@ import layout from '../templates/components/empty-state';
  *
  * @param title=null{String} - A short label for the empty state
  * @param message=null{String} - A description of why a user might be seeing the empty state and possibly instructions for actions they may take.
+ * @param icon=null{String} - A optional param to display icon to the right of the title
```

Comment:
this param should be in brackets to indicate that it's optional and should have the default value of `''`. see the example JSDOCs in the readme here:

https://github.com/hashicorp/vault/blob/31bbb813e7b8250a1e9ffe09ddc81e8f2653b476/ui/README.md#L150